TrafficCapture/dockerSolution/src/main/docker/migrationConsole/lib/console_link/tests/test_cli.py

```diff
@@ -66,6 +67,40 @@ def test_cli_with_backfill_describe(runner, env, mocker):
     assert result.exit_code == 0
 
 
+def test_cli_snapshot_create(runner, env, mocker):
+    mock = mocker.patch('console_link.logic.snapshot.create')
```

Comment:
the disadvantage of mocking this point instead is that we don't have test coverage for anything in the model, but I'm okay with improving this later.